optimum/exporters/tasks.py

```diff
@@ -1560,7 +1560,9 @@ def _infer_task_from_model_name_or_path(
             library_name = TasksManager.infer_library_from_model(model_name_or_path, subfolder, revision)
 
             if library_name == "diffusers":
-                class_name = model_info.config["diffusers"]["class_name"]
+                class_name = model_info.config["diffusers"].get("class_name", None) or model_info.config[
+                    "diffusers"
+                ].get("_class_name", None)
```

Comment:
```suggestion
                if hasattr(model_info.config["diffusers"], "class_name"):
                    class_name = model_info.config["diffusers"]["class_name"]
                elif hasattr(model_info.config["diffusers"], "_class_name"):
                    class_name = model_info.config["diffusers"]["_class_name"]
                else:
                    raise ValueError(f"Could not automatically infer the task name for {model_name_or_path}. Please open an issue at https://github.com/huggingface/optimum/issues.")
```

Reply:
@fxmarty `model_info.config["diffusers"]` is a dictionary and `class_name` or `_class_name` are keys instead of attributes.

Reply:
feel free to edit accordingly

Reply:
Done!

Reply:
@JingyaHuang Is the suggestion fine as is in the end?

Reply:
It's good.